{"uuid":"b1dbb63d-dc2a-485f-9569-09d49e09338d","name":"Scenario Outline: Login with invalid credentials","children":["6cc0ce2b-c7cd-4bdd-a120-a6c7b83a239a"],"befores":[{"name":"StepDefs.Hooks.beforeScenario()","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1745430987672,"stop":1745430989284}],"afters":[{"name":"StepDefs.Hooks.afterScenario()","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER is failed: The following asserts failed:\n\texpected [2] but found [1]","trace":"java.lang.AssertionError: The following asserts failed:\n\texpected [2] but found [1]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat StepDefs.Hooks.afterScenario(Hooks.java:28)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1745431018730,"stop":1745431020499}],"start":1745430987672,"stop":1745431020500}